lte/gateway/c/oai/tasks/nas/nas_state_converter.cpp

```diff
@@ -1581,6 +1605,13 @@ void NasStateConverter::emm_context_to_proto(
   ue_network_capability_to_proto(
       &state_emm_context->_ue_network_capability,
       emm_context_proto->mutable_ue_network_capability());
+  nas_timer_to_proto(
```

Comment:
No need to read/write the timer itself as timer IDs are not the same after service restart. 

ToDo: Need to remove logic that writes other NAS timers to state

Reply:
Saving only timer value